otx/api/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,4 +1,4 @@
 openvino==2022.3.0
 openmodelzoo-modelapi==2022.3.0
-otx @ git+https://github.com/openvinotoolkit/training_extensions/@dd03235da2319815227f1b75bce298ee6e8b0f31#egg=otx
+otx @ git+https://github.com/openvinotoolkit/training_extensions/@8c11c3d42c726e6e0eda7364f00cf8ed4dbdc2e9#egg=otx
```

Comment:
@goodsong81 The previous and updated commit is not a part of OTX develop, but from the fork. Can you please clarify if we need to update this commit again, then these changes will be merged.

Reply:
As far as pip install works, it's OK as we'll anyway update this commit to a PyPI version at the time of release.
![image](https://user-images.githubusercontent.com/4629606/231624631-3a12eab2-4051-49f7-8744-4da7d1f3b4a7.png)